README.md

```diff
@@ -544,7 +544,7 @@ krane render -f ./path/to/template/dir/template.yaml.erb > template.yaml
 - `--bindings=BINDINGS`: Makes additional variables available to your ERB templates. For example, `krane render --bindings=color=blue size=large -f some-template.yaml.erb` will expose `color` and `size` to `some-template.yaml.erb`.
 - `--current-sha`: Expose SHA `current_sha` in ERB bindings
 
-You can add additional variables using the `--bindings=BINDINGS` option which can be formated as a string, JSON string or path to a JSON or YAML file. Complex JSON or YAML data will be converted to a Hash for use in templates. To load a file the argument should include the relative
+You can add additional variables using the `--bindings=BINDINGS` option which can be formatted as a string, JSON string or path to a JSON or YAML file. Complex JSON or YAML data will be converted to a Hash for use in templates. To load a file the argument should include the relative
 file path prefixed with an `@` sign. An argument error will be raised if the string argument cannot be parsed, the referenced file does not include a
  valid extension (`.json`, `.yaml` or `.yml`) or the referenced file does not exist.
```

Comment:
- >To load a file the argument should include... -> To load a file, the argument... (missing comma)

- The line breaks here are quite strange, can we clear those up while we're here, too?

Reply:
I added the missing coma and fixed the line breaks.